ios/RNGoogleMobileAds/RNGoogleMobileAdsAppOpenModule.h

```diff
@@ -23,7 +23,6 @@
 
 @interface RNGoogleMobileAdsAppOpenModule : NSObject <RCTBridgeModule>
 
-@property(strong, nonatomic) GADAppOpenAd* appOpenAd;
-@property(strong, nonatomic) RNGoogleMobileAdsFullScreenContentDelegate* appOpenDelegate;
++ (void)removeAppOpen;
```

Comment:
Is that correct? Is the only thing left in the header really the `removeAppOpen` symbol? If so, there is not an unneeded import I think of RNGoogleMobileAdsFullScreenContentDelegate, if not, does this work in local testing?